server/src/routes/api/v1/showRouter.js

```diff
@@ -0,0 +1,25 @@
+import express from "express"
+// import { Show } from 
+
+const showsRouter = new express.Router()
+
+showsRouter.get("/:id/info", async(req,res) => {
+  const showId = req.params.id
+  try{
+    const show = await Show.query().findById(showId)
+    return res.status(200).json({show})
+  }catch(err){
+    return res.status(500).json({errors: err})
+  }
+})
+
+showsRouter.get("/", async(req,res) => {
+  try{
+    const showList = await Shows.query()
+    return res.status(200).json({showList})
+  }catch(err){
+    return res.status(500).json({errors: err})
+  }
+})
```

Comment:
this endpoint is not being used by any code in this branch, and should be removed. Allow the indexpage group to focus on this endpoint